sources/packages/api/src/route-controllers/institution-locations/institution-location.aest.controller.ts

```diff
@@ -0,0 +1,39 @@
+import { Controller, Get, Param } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { ClientTypeBaseRoute } from "../../types";
+import BaseController from "../BaseController";
+import { InstitutionLocationControllerService } from "./institution-location.controller.service";
+import { InstitutionLocationAPIOutDTO } from "./models/institution-location.dto";
+
+/**
+ * Institution location controller for institutions Client.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("institution/location")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-institution`)
```

Comment:
shouldn't the tag also have `location`?

Reply:
I believe as institution locations falls under institution(institution/location), we are keeping it under institution apitag.
how are we doing for others?

Reply:
May be this a point that we can discuss @andrewsignori-aot  @guru-aot @henryoforeh-aot. This story is the right time for that.

Reply:
My understanding right now was to just copy the content from the `@Controller` and append it to the `@ApiTags`, as below:
```typescript
@ApiTags(`${ClientTypeBaseRoute.AEST}-institution/location`)
```

Reply:
I believe the location tag should be also available as other DEVS suggested.